apps/concierge_site/assets/js/feedback-form.js

```diff
@@ -39,8 +39,13 @@ const handlefeedbackFormSubmit = feedbackFormEl => e => {
         alertEl = makeAlert(
           "feedback-alert",
           "alert-success",
-          "Thanks for your feedback!"
+          ""
         );
+        alertEl.innerHTML = `<p>Thanks for your feedback!</p>
+          <p> If you're interested in continuing to share your experiences you can sign up at
+            <a href="http://mbta.com/ridertechfeedback"> mbta.com/ridertechfeedback</a> to be contacted in the future.
```

Comment:
```suggestion
            <a href="http://mbta.com/ridertechfeedback">mbta.com/ridertechfeedback</a> to be contacted in the future.
```

nitpick: Extra space in the link.